src/Blog.jsx

```diff
@@ -0,0 +1,14 @@
+import './Blog.css';
+import React, { Component, Fragment } from 'react';
+
+import { Post } from './Post';
+
+export class Blog extends Component {
+  render() {
+    return (
+      <Fragment>
+        <Post postItems={this.props.postItems} />
```

Comment:
```suggestion
        {this.props.postItems.map((post, index) => (<Post key={index} {...this.props.postItems} />))
```
Было бы правильнее сделать так, чтобы Post хранил в себе только логику отображения (т.е. только JSX и это был бы чисто презентационный компонент). Т.е. он, исходя из своего названия, генерировал бы только один пост:) Так компонент будет более гибким